crates/transaction-pool/src/validate/eth.rs

```diff
@@ -482,11 +488,16 @@ where
         if self.chain_spec.is_prague_active_at_timestamp(new_tip_block.timestamp) {
             self.fork_tracker.prague.store(true, std::sync::atomic::Ordering::Relaxed);
         }
+
+        if new_tip_block.gas_limit() > 0 {
```

Comment:
This check might not be necessary since new_tip_block.gas_limit() will never be 0 for a valid block but i added it for sanity check, let me know what you think.

Reply:
yeah we can remove this imo